tests/zfs-tests/tests/functional/events/zed_cksum_reported.ksh

```diff
@@ -0,0 +1,122 @@
+#!/bin/ksh -p
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or https://opensource.org/licenses/CDDL-1.0.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2022 by Rob Wing <rob.wing@klarasystems.com>
```

Comment:
I feel like this should be a Klara Systems copyright notice, but we should ask @allanjude about that.

Reply:
@rob-wing if you could just address this feedback this PR should be good to go.

Reply:
@ryao, thanks for pointing that out - I've updated the copyright notice accordingly.